src/routes/auth.js

```diff
@@ -455,7 +454,7 @@ router.get('/verifyRToken', async (req, res) => {
             });
         }
 
-        verify(requestToken, client.access_token, {
+        verify(requestToken, keys.privateKey, {
             algorithms: ['HS256'],
         });
```

Comment:
Change this to `keys.publicKey`, and algorithms to `['RS256']`. This is because the token was originally signed by CASI's private key and should be verified by its public key.

[Reference](https://github.com/devclub-iitd/SingleSignOn/blob/3358d88abdaa7202a5921609fb8009b345c0c209/src/utils/utils.js#L347)